src/DataFormatter.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DATAFORMATTER_H
+#define DATAFORMATTER_H
+
+#include <QLocale>
+#include <QVariant>
+#include <QDate>
+#include <QDateTime>
+#include <QDebug>
+
+class DataFormatter
+{
+public:
+    DataFormatter(QString delimiter = ";", QString quoteSymbol = "\"", bool alwaysQuoteStrings = true, bool replaceNewLine = true, QHash<QString, QString> formatOverrides = QHash<QString, QString>());
```

Comment:
I don't think delimiter and quote belong here, as that is more a concern for the exporter based on the file type (ex: json / xml would do it differently).

Reply:
I kinda wanted to avoid having string handling function in Csv, but it does belong there.
Will update it.